build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
I think Windows (maybe Mas OS) guys won't be happy.
Or maybe it's ok with wsl?

Reply:
For windows users, we could write an equivalent script